homeassistant/components/wallbox/__init__.py

```diff
@@ -72,10 +71,10 @@ def _get_data(self):
                 CONF_MAX_CHARGING_CURRENT_KEY
             ]
 
-            filtered_data = {k: data[k] for k in CONF_SENSOR_TYPES if k in data}
+            filtered_data = {k: data[k] for k in SENSOR_TYPES if k in data}
 
             for key, value in filtered_data.items():
-                if (sensor_round := CONF_SENSOR_TYPES[key][CONF_ROUND]) is not None:
+                if (sensor_round := SENSOR_TYPES[key].precision) is not None:
```

Comment:
Rounding can be done in the platforms as needed if the description has that attribute.